packages/common/src/configuration/service-configuration.ts

```diff
@@ -35,6 +35,7 @@ export interface ScanRunTimeConfig {
 export interface RestApiConfig {
     maxScanRequestBatchCount: number;
     scanRequestProcessingDelayInSeconds: number;
+    maxScanRequestWaitTimeInSeconds: number;
```

Comment:
NIT: This does not look like a config for rest api.

Reply:
it is not. we can move it out later.